src/cdk/a11y/live-announcer/live-announcer.ts

```diff
@@ -36,7 +38,9 @@ export class LiveAnnouncer implements OnDestroy {
   constructor(
       @Optional() @Inject(LIVE_ANNOUNCER_ELEMENT_TOKEN) elementToken: any,
       private _ngZone: NgZone,
-      @Inject(DOCUMENT) _document: any) {
+      @Inject(DOCUMENT) _document: any,
+      @Optional() @Inject(LIVE_ANNOUNCER_DEFAULT_OPTIONS)
+          private _defaultOptions?: LiveAnnouncerDefaultOptions) {
```

Comment:
Wrong indentation here?

Reply:
It's intentional. I had to wrap the line so it doesn't go over the limit.

Reply:
Yeah the wrapping makes sense. Though the additional spaces are inconsistent with what we usually do. Not trying to be too nitpicky, but consistency is a good practice 😄 

e.g. see `drag.ts`

https://github.com/angular/material2/blob/d69bc45ed156ad9f5fd2f2e062fc10112d2ea81e/src/cdk/drag-drop/directives/drag.ts#L168-L169